docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+   ...
+   ```
+
+5. Scaffold a LoopBack 4 application in the current directory with:
+
+   ```sh
+   appsody init nodejs-loopback
+   ```
+
+   You should see output like this:
+
+   ```
+   Checking stack requirements...
+   Skipping Buildah - No requirements set.
+   Skipping Docker - No requirements set.
+   Skipping Appsody - No requirements set.
+   Running appsody init...
+   Downloading nodejs-loopback template project from https://github.com/appsody/stacks/releases/download/nodejs-loopback-v0.1.7/incubator.nodejs-loopback.v0.1.7.templates.scaffold.ta
+   r.gz
+   Download complete. Extracting files from /appsodyLB4Todo/nodejs-loopback.tar.gz
+   Setting up the development environment
+   Your Appsody project name has been set to appsodylb4todo
+   Pulling docker image appsody/nodejs-loopback:0.1
+   Running command: docker pull appsody/nodejs-loopback:0.1
+   0.1: Pulling from appsody/nodejs-loopback
+   ...
+   Status: Downloaded newer image for appsody/nodejs-loopback:0.1
+   docker.io/appsody/nodejs-loopback:0.1
+   [Warning] The stack image does not contain APPSODY_PROJECT_DIR. Using /project
+   Running command: docker run --rm --entrypoint /bin/bash appsody/nodejs-loopback:0.1 -c find /project -type f -name .appsody-init.sh
+   Successfully initialized Appsody project with the nodejs-loopback stack and the default template.
+   ```
+
+   The most important lines to make note of are:
+
+   ```
+   Your Appsody project name has been set to appsodylb4todo
+
+   and
+
+   Successfully initialized Appsody project with the nodejs-loopback stack and the default template.
+   ```
+
+6. Install all the dependencies of this application with:
+
+   ```sh
+   npm install
+   ```
+
+7. Now that all the dependencies have been installed, it is a perfect time to
+   open your favourite IDE editor on this project.
+
+   In my case, I using VS Code.
+
+   ```sh
+   code .
+   ```
+
+8. The basic structure of the application looks like this:
+
+   ![lb4appsody_template_files_1.png](./imgs/lb4appsody_template_files_1.png)
+
+   One important file is the Appsody configuration file for your project named
+   **.appsody-config.yaml** .
+
+   It defines the **name** of your project, and the **stack** (template) on
+   which it is based.
+
+   ```yaml
+   project-name: appsodylb4todo
+   stack: appsody/nodejs-loopback:0.1
+   ```
+
+9. The source code of the LoopBack 4 application is located in the **src**
+   directory.
+
+   ![lb4appsody_template_files_2.png](./imgs/lb4appsody_template_files_2.png)
+
+10. This basic application only defines one API endpoint `'/ping'` in the file
+    **src/controllers/ping.controller.ts**
+
+In the next section, we will add more API endpoints.
+
+## Building Your Application
+
+The template application is only a starting point. Let's build a proper, basic
+LoopBack 4 application.
+
+Let's build the
```

Comment:
I wonder if we need this section because it's the same as the todo tutorial. 
I can see the advantage of having all the instructions in one place without clicking other docs. However, my concern of having duplicating the content here is if there's any change in the prompts, we need to update it in 2 places.  Personally, I'd like to see this tutorial to be more focus on the specific things we need to do when deploying Apposody.

Reply:
Todo also talks about the geoservice. We don't need that. Also. The todo doc is out of date in terms of prompts. We can replace this `Building your application` section with `follow the Todo tutorial starting with 'Add model' and do not do 'Adding geo-service'. 

I was trying to avoid any relational database stuff because I wanted to avoid talking about setting up a database on IBM Cloud.

Raymond's changes to the structure/documentation of the shopping cart application was geared to show how to deploy an app docker container, a mongo container, and a redis container to a kubernetes cluster. So I figured this was sufficient.

Reply:
With Appsody, using the nodejs-loopback stack ( a template of a loopback app), it is very simple to run/debug/test/stop and app locally. And very simple to deploy to kubernetes. Before deploying to kubernetes, we would need to change the db url in the datasource json.

Reply:
@dhmlau , we can start with this, and then provide a follow-up in the future for dealing with a relational database

Reply:
sure. as the first step, I'm fine with just using the in-memory database.